pkg/kubelet/dockershim/libdocker/fake_client.go

```diff
@@ -657,10 +656,10 @@ func (f *FakeDockerClient) PullImage(image string, auth dockertypes.AuthConfig,
 			return ImageNotFoundError{ID: image}
 		}
 
-		authJson, _ := json.Marshal(auth)
+		authJSON, _ := json.Marshal(auth)
```

Comment:
I think our style guide is to use all caps for acronyms. We have generated code that don't follow this convention, but otherwise, we should still follow that.